games-emulation/mupen64plus-core/mupen64plus-core-2.5.9-r4.ebuild

```diff
@@ -42,6 +42,7 @@ BDEPEND="
 "
 
 PATCHES=(
+	"${FILESDIR}"/${P}-debugger-fix-binutils-2.39-compat.patch
```

Comment:
Could you link the upstream commit in a comment? I know we failed to do this in the past but let's improve.

Reply:
There is no upstream commit for this yet.

Reply:
I have updated the commit with the PR link.

Reply:
Sorry, I meant in a comment above the entry in `PATCHES`, so that anyone reading the ebuild would see it.